microsoft-365/enterprise/ms-cloud-germany-transition-add-experience.md

```diff
@@ -30,6 +30,8 @@ The following sections provide additional information on customer experiences wh
 
 ## Services
 
+Between Phase 2 of 9 and Phase 3 of 9, Partner Portal may not be accessible. During this time, Partner may not be able to access the tenants information on the Partner Portal. Since each migration is different, the duration of in-accessibility could be in hours. 
```

Comment:
```suggestion
Between Phase 2 of 9 and Phase 3 of 9, Partner Portal may not be accessible. During this time, the partner may not be able to access the tenant's information on the Partner Portal. Since each migration is different, the duration of in-accessibility could be a number of hours. 
```